tmcprototype/CspSubarrayLeafNode/CspSubarrayLeafNode/CspSubarrayLeafNode.py

```diff
@@ -26,20 +30,23 @@
 from skabase.SKABaseDevice.SKABaseDevice import SKABaseDevice
 # Additional import
 # PROTECTED REGION ID(CspSubarrayLeafNode.additionnal_import) ENABLED START #
+from future.utils import with_metaclass
 import CONST
 import json
 # PROTECTED REGION END #    //  CspSubarrayLeafNode.additionnal_import
 
 __all__ = ["CspSubarrayLeafNode", "main"]
 
-
-class CspSubarrayLeafNode(SKABaseDevice):
+class CspSubarrayLeafNode(with_metaclass(DeviceMeta, SKABaseDevice)):
```

Comment:
Out of curiosity, why do we require a metaclass definition? Did we face any issues without it? It was not there previously.

Reply:
This change had happened when we migrated our code from P2.7 to P3.5.
We don't face any issue but our code is now in P3.5. Hence, it should be kept like that.